### ISSUE TITLE
Status 400 received when requesting transcript

### ISSUE THREAD

**Issue by beirne** [lstrzepek/obsidian-yt-transcript#21]
### Discussed in https://github.com/lstrzepek/obsidian-yt-transcript/discussions/20

<div type='discussions-op-text'>

<sup>Originally posted by **beirne** June  4, 2023</sup>
I just installed the plugin into Obsidian on my Mac but it can't pull transcripts. I get a status 400 error. This happens with videos that have both auto-generated and creator-generated transcripts.

<img width="260" alt="image" src="https://github.com/lstrzepek/obsidian-yt-transcript/assets/6078590/50905047-6596-4239-b14b-e7b56e0c5349">

I'm running YTranscript 0.7.1 on Obsidian 1.3.5 on MacOS 13.4.</div>

**Comment by beirne** [comment]
I get the status 400 with 0.8.1 as well, both from a selected and an entered URL

**Comment by lstrzepek** [comment]
Thanks for your feedback! 
Please add Yt link which makes you trouble. I'm also getting 400 but on mobile but still have no clue how to debug there.

**Comment by beirne** [comment]
Here's a link I just tried and failed with. https://youtu.be/FatnXnlwAc4
Here how I tried doing it. They all failed except for the one time I say how it worked.
- Pasted link into a note, selected it, opened the command palette, and picked the choice for using the selection.
- Created a hotkey for using ytranscript with the selection, and did the previous sequence with it instead of the command palette.
-     From the command palette asked to be prompted for the URL. It worked once, but after that it went back to status 400.
